sgv2-docsapi/src/main/java/io/stargate/sgv2/docsapi/service/schema/JsonSchemaManager.java

```diff
@@ -55,7 +54,7 @@ public Uni<JsonNode> getJsonSchema(Uni<Schema.CqlTable> table) {
             t -> {
               String comment = t.getOptionsMap().getOrDefault("comment", null);
               if (comment == null) {
-                return null;
+                throw new ErrorCodeRuntimeException(ErrorCode.DOCS_API_JSON_SCHEMA_DOES_NOT_EXIST);
```

Comment:
this is bad now.. every time you try to validate a document and here is no schema, an exception will be raised and this is extra overhead in order to collect the stack trace and everything.. `null` was good.. Please revert this in on our write path..